benchmarking/commons/hpo_main_simulator.py

```diff
@@ -161,8 +161,8 @@ def parse_args(
                 name="fcnet_ordinal",
                 type=str,
                 choices=("none", "equal", "nn", "nn-log"),
-                default="none",
-                help="Ordinal encoding for fcnet categorical HPs",
+                default="nn-log",
```

Comment:
This is a change of defaults. Before, the fcnet `choice` HP would remain. Now, it is by default replaced by `ordinal`.

Reply:
Would this require any of our customers to change their code? 

This doesn't really matter today since we are still pre-v1.0.0, but thinking ahead to when we do mark our API as stable and want to avoid breaking changes.

Reply:
Except for myself, I don't think anybody is using this right now. It is only now properly documented.